microsoft-edge/devtools/network/reference.md

```diff
@@ -155,21 +174,32 @@ To capture a screenshot:
  
 1. Press **Esc** to close the screenshot viewer.
 
-<!--  ### Replay XHR request  -->
 
-<!--  To replay an XHR request, right-click the request in the Requests table, and then click **Replay XHR**.  -->
+<!-- ------------------------------ -->
+#### Replay XHR request
+<!-- https://developer.chrome.com/docs/devtools/network/reference/#replay-xhr -->
+
+To replay an XHR request, do one of the following in the **Requests** table:
+
+* Select the request and press **R**.
+* Right-click the request and select **Replay XHR**.
+
+<!-- For example, the [Network tutorial](https://microsoftedge.github.io/Demos/network-tutorial/) demo page: -->
 
 <!--
-![Click Replay XHR](../media/network-replay-xhr.png)
+![Click Replay XHR](todo png: selecting-replay-xhr.png)
+todo: need demo page that has XHR; else no Replay XHR menuitem
```

Comment:
need demo page that has XHR
@captainbrosset 
not sure if discussed

Reply:
https://microsoftedge.github.io/Demos/devtools-network-reference/ has XHR button